docs/blog/2018-06-26-card-sort-results/index.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Presenting the Docs Redesign Card Sort Results
+date: 2018-06-26
+author: Shannon Soper
+tags: ["documentation"]
+---
+
+## What is a card sort?
+
+Recently, 36 Gatsby users completed an open card sort to help make [next.gatsbyjs.org](https://next.gatsbyjs.org/) easier to use. Without any outside input or help, they each sorted 90 cards into categories that made sense to them and then named those categories.
+
+### Raw & beautified data
+
+I ran this card sort through [OptimalSort](https://www.optimalworkshop.com/), a software that moderates the card sort and presents the data in usable forms. If you’re curious to see the full results, you can [view the data](https://www.optimalworkshop.com/optimalsort/x87kpp82/5x34psa3-0/shared-results/fa8b66knb66qyhwh5l8j38bd273vkkm7), including raw data as well as things like a similarity matrix and a [dendogram](https://support.optimalworkshop.com/hc/en-us/articles/201997650-Interpreting-the-OptimalSort-dendrograms).
+
+## How did participants categorize the docs?
+
+Most people sorted the docs into the following categories (I included multiple names when the some category names were equally popular):
+* Get Started / Welcome to Gatsby / Intro
+* Development environment
+* Releases & Migration
+* Core Concepts / About Gatsby
+* Advanced Guides
+* API Reference
+* Tutorials / Examples
+* Contributing
+
+These categories weren’t too surprising. However, upon further digging into the data, I discovered some unexpected results that further refine how we might categorize the docs and even write new docs.
```

Comment:
categorize the docs, write new docs, and change how we think about and talk about Gatsby.